pkg/kubelet/kubelet_volumes.go

```diff
@@ -107,18 +110,29 @@ func (kl *Kubelet) cleanupOrphanedPodDirs(
 		// If there are still volume directories, do not delete directory
 		volumePaths, err := kl.getPodVolumePathListFromDisk(uid)
 		if err != nil {
-			glog.Errorf("Orphaned pod %q found, but error %v occurred during reading volume dir from disk", uid, err)
+			orphanVolumeErrors = append(orphanVolumeErrors, fmt.Errorf("Orphaned pod %q found, but error %v occurred during reading volume dir from disk", uid, err))
```

Comment:
i think a while back it also used to spam this if  there were any volume paths floating around